cycling_quality_index.py

```diff
@@ -2144,9 +2165,9 @@ def addDelimitedValue(var, value):
 
         layer.updateFields()
 
-    #clean up data set
+    # clean up data set
     print(time.strftime('%H:%M:%S', time.localtime()), 'Clean up data...')
-    processing.run('native:retainfields', { 'INPUT' : layer, 'FIELDS' : retained_attributes_list, 'OUTPUT': dir_output })
+    processing.run('native:retainfields', { 'INPUT' : layer, 'FIELDS' : retained_attributes_list, 'OUTPUT': f"{dir_output}" })
```

Comment:
das muss als f-string formattiert sein, da ein `pathlib.Path` kein string ist, und nicht implizit in einen verwandelt werden kann. ich hätte auch `... + str(dir_output) + ...` nehmen können, f-strings sind aber besser